src/test/groovy/sample/SampleJobSpec.groovy

```diff
@@ -0,0 +1,93 @@
+package sample
+
+import groovy.util.slurpersupport.GPathResult
+import groovy.util.slurpersupport.Node
+import javaposse.jobdsl.dsl.DslScriptLoader
+import javaposse.jobdsl.dsl.JobManagement
+import javaposse.jobdsl.plugin.JenkinsJobManagement
+import javaposse.jobdsl.dsl.GeneratedItems
+import javaposse.jobdsl.dsl.GeneratedJob
+import org.junit.ClassRule
+import org.jvnet.hudson.test.JenkinsRule
+import spock.lang.Shared
+import spock.lang.Specification
+import spock.lang.Unroll
+
+/**
+ * Basic POC tests for the sample dsl script sample/jobs/sampleJob.groovy
+ */
+class SampleJobSpec extends Specification {
+
+    @Shared
+    @ClassRule
+    JenkinsRule jenkinsRule = new JenkinsRule()
+
+    @Unroll
```

Comment:
@benpatterson you can't really see the benefit of it here, but if you use something like a data table or iterate over instances within a single test, the @Unroll annotation will report them as separate tests. I was originally doing that and forgot to pull them out. I will add an example showing this off